.github/workflows/config/gitlab_commits.txt

```diff
@@ -1,2 +1,2 @@
 nvidia-mgpu-repo: cuda-quantum/cuquantum-mgpu.git
-nvidia-mgpu-commit: 1ca225d25811732198a5533f62a3858387a2f866
+nvidia-mgpu-commit: 331c60e52130402c896a66e3af925c236de876a7
```

Comment:
AFAICT, this commit is not on the `main` branch. Is this something that would be merge to main and then we would follow-up with another to commit to GH once it is on main?

Reply:
The commit here, yes. Given that I had some difficulties getting the full build here with the runners issue and this doing a full cache update that gets kicked out, I'd prefer to pull this in when the cache exists, and then make the small update to update the commit hash with the correct one. Thien and I have been coordinating the libcpp branch, so I believe that one is just a formality to merge.

Reply:
The changes on mgpu are now on main, including the synchronization change, and this commit now picks up both those changes.